crates/pessimistic-proof-core/src/local_state/mod.rs

```diff
@@ -254,21 +295,18 @@ impl NetworkState {
                     l1_info_root: multi_batch_header.l1_info_root,
                     origin_network: multi_batch_header.origin_network,
                     aggchain_params: *aggchain_params,
-                    commit_imported_bridge_exits: commit_imported_bridge_exits(
-                        multi_batch_header
-                            .imported_bridge_exits
-                            .iter()
-                            .map(|(exit, _)| exit.global_index),
-                    ),
+                    commit_imported_bridge_exits: imported_hash,
                 };
 
                 sp1_zkvm::lib::verify::verify_sp1_proof(
                     aggchain_vkey,
                     &aggchain_proof_public_values.hash().into(),
                 );
+
+                PPRootVersion::V2
```

Comment:
Shouldn't it be `v3`?

Reply:
same as for https://github.com/agglayer/agglayer/pull/689#discussion_r1987184945